anime/Services/ImportScheduleService.php

```diff
@@ -14,8 +14,14 @@ class ImportScheduleService implements Service {
 
     // Initializes the service with |$options|, defined in the website's configuration file.
     public function __construct(array $options) {
-        $required = ['frequency', 'identifier', 'volunteers', 'program', 'mapping', 'schedule',
-                     'destination_shifts', 'destination_program'];
+        $required = ['frequency', 'identifier', 'volunteers', 'mapping', 'schedule',
+                     'destination_shifts', 'destination_team_program'];
+
+        // Backwards compatibility
+        if (array_key_exists('destination_program', $options)) {
+            $options['destination_team_program'] = $options['destination_program'];
+        }
```

Comment:
It's fine to not maintain backwards compatibility, there's really only one consumer of this system and that's us :P